_data/simple-icons.json

```
@@ -13794,6 +13794,16 @@
             "source": "https://www.streamlit.io/brand",
             "guidelines": "https://www.streamlit.io/brand"
         },
+        {
+            "title": "StreamRunners",
+            "hex": "6644F8",
+            "source": "https://streamrunners.fr/index",
```

For better future-proofing, the `/index` can be removed here.
```suggestion
            "source": "https://streamrunners.fr",
```

---

Hi Peter!
Regarding this, it appears that if the user is connected, he's not redirected on the index page but on another main page of the website. But in both cases, the logo is present. If it's not an issue, we can effectively remove the `/index`!

---

As long as the "[Visuals](https://drive.google.com/drive/folders/1vQOIHyfmzuWIYGYg-_mbJDif5eDDMt2b?usp=sharing)" link in the footer is available on both pages then the `/index` can be removed, yes. If it's _not_ available on the other main page, though, then it should be kept.